src/delete-uniswap/sdk.ts

```diff
@@ -52,8 +53,8 @@ export default class Sdk {
     const route = new Route([pair], token);
     const price = route.midPrice.toSignificant(6);
 
-    // console.log('inv', route.midPrice.invert().toSignificant(6)) // 0.00496756
-    // console.log('price', price) // 201.306
+    // logger.debug('inv', route.midPrice.invert().toSignificant(6)) // 0.00496756
+    // logger.debug('price', price) // 201.306
```

Comment:
I assume this comments serve as documentation? but they will break if we try to use them. I'll add the dependency.

Reply:
Thanks man